openapi-cli/src/test/resources/ballerina-to-openapi/headers/header_scenario11.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/http;
+
+listener http:Listener helloEp = new (9090);
+
+service /payloadV on helloEp {
+    string y = "hello";
+    resource function get ping(@http:Header int headerValue) returns http:Ok {
```

Comment:
Cant we have a single test source with all the possible header types (i.e. simple types, nillable types, array types) and validate instead of having single test source for each header type?

Reply:
move to one test case with similar scenarios to handle int and boolean type :)